docs/3.0/actions.md

```diff
@@ -362,6 +362,47 @@ end
 ```
 :::
 
+:::option `action icons`
```

Comment:
Other labels have only the option name. Theis would be `icon`

Reply:
@adrianthedev the other options on actions are response types I changed the icon an divider location in this PR https://github.com/avo-hq/avodocs/pull/187

I couldn't commit to this one

Reply:
I get that, but I think we should do the same with these arguments.

Maybe in a docs reorganization?

Reply:
Yes agree, just created this issue to track it https://github.com/avo-hq/avodocs/issues/195